dom0/sd-decrypt.sls

```diff
@@ -36,11 +36,15 @@ qvm-tags sd-decrypt add sd-decrypt-vm:
   cmd.run
 
 # Allow dispvms based on this vm to use sd-gpg
-sed -i '1i$tag:sd-decrypt-vm sd-gpg allow' /etc/qubes-rpc/policy/qubes.Gpg:
-  cmd.run:
-  - unless: grep -qF '$tag:sd-decrypt-vm sd-gpg allow' /etc/qubes-rpc/policy/qubes.Gpg
+/etc/qubes-rpc/policy/qubes.Gpg:
+  file.line:
+    - content: $tag:sd-decrypt-vm sd-gpg allow
+    - mode: insert
+    - location: start
 
 # Allow sd-decrypt to open files in sd-svs
-sed -i '1isd-decrypt sd-svs allow' /etc/qubes-rpc/policy/qubes.OpenInVM:
-  cmd.run:
-  - unless: grep -qF 'sd-decrypt sd-svs allow' /etc/qubes-rpc/policy/qubes.OpenInVM
+/etc/qubes-rpc/policy/qubes.OpenInVM:
```

Comment:
In Salt, top-level task titles, which are the top key of a YAML dict, must be _globally_ unique across the Salt config. That means we cannot reuse the filename as the title, since we modify this same file for different reasons across the Salt config (as shown in the diff you're presenting here).

Let's instead default to a reasonable pragma for naming these types of task intuitively: `<vm_name>-[dom0-]<filename>`. So in this case: `sd-decrypt-dom0-qubes.OpenInVM`. The entire task would then look like:

```
sd-decrypt-dom0-qubes.OpenInVM:
  file.line:
    - name: /etc/qubes-rpc/policy/qubes.OpenInVM
    - mode: insert
    - location: start
```

Similar changes should be made to the other `file.line` conversions presented here.